src/query/sql/src/planner/semantic/lowering.rs

```diff
@@ -261,6 +261,10 @@ impl ScalarExpr {
                 let scalar = &udf.scalar;
                 scalar.as_raw_expr()
             }
+
+            ScalarExpr::AsyncFunctionCall(_table_func) => {
+                unreachable!()
```

Comment:
This will cause `select nextval(seq)` panic, do we plan to support this feature in the future?